src/components/Utils/settings/getCreateInventorySetting.js

```diff
@@ -1,6 +1,6 @@
 import { get } from 'lodash';
```

Comment:
basically, I don't see too much sense of such deep nesting and changing all imports for `getCreateInventorySetting.js `.
If you want such nesting, you can export required function from `Utils/index.js`. 
But in case you're moving the code - maybe it's better to move it under `src/common/utils`

Reply:
Updated